docs/start/framework/installation.md

```diff
@@ -11,6 +11,14 @@ Most projects start with a template. Let's use a basic template maintained by Re
 npx create-react-router@latest my-react-router-app
 ```
 
+You can also create a project using one of the many available [deployment templates](https://github.com/remix-run/react-router-templates), which will configure your project for the hosting provider.
+
+```shellscript nonumber
+npx create-react-router@latest my-node-default-react-router-app --template remix-run/react-router-templates/default
+```
+
+The other available template options are `remix-run/react-router-templates/node-custom-server`, `remix-run/react-router-templates/cloudflare`, `remix-run/react-router-templates/vercel`, and `remix-run/react-router-templates/netlify`. 
+
```

Comment:
Thanks for including this. Let's move this down below where we already talk about the templates

Also, I think we can avoid enumerating all the templates, since it's something we'll have to keep in sync between here and the `react-router-templates` repo, which means it'll inevitably get out of sync

Reply:
Thanks! At the moment we're going to keep the docs simple and make it more clear how you can use the CLI. Since the templates are linked to, we'd prefer to direct users to that repository, since that's where issues, PRs, instructions, etc. live

Once we have a deployment doc that may be a better place for us to enumerate all the options we have inside of our docs. The main thing is we prefer to keep the installation doc really simple and to the point. Vite has a different purpose with their docs than we do